### PR TITLE
Updated tollFreeWildCardPattern and quantity for search order requests

### PR DESCRIPTION
**Pull request by jmulford-bw**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,3 +1,4 @@
+dist: trusty
```
Comment (dtolb) [inline .travis.yml]: What's this do?

File: README.md [hunk 2]
```diff
@@ -2,6 +2,12 @@
 
 Java SDK for Bandwidth's number management API.
 
+## Release Notes
+
+| Version | Notes |
+|:---|:---|
+| 0.6.0  | Fixed `quantity` and `tollFreeWildCardPattern` for search requests |
```
Comment (NathanNorman) [inline README.md]: How'd we get to `0.6.0`? I thought we had only released 0.1.0 previously.

Comment (jmulford-bandwidth) [reply README.md]: oof. You're right. I was looking at the messaging sdk version on accident

Comment (jmulford-bandwidth) [reply README.md]: Fixed 🎉

**Pull request merged**